{"name": "test_empty[hcodes11]", "status": "failed", "statusDetails": {"message": "AssertionError: assert [('00150', 'У... 3, ...), ...] == []\n  Left contains 17043 more items, first extra item: ('00150', 'Удельный показатель по отказам 1, 2 категории', 'ед/млн. поездо-км', 480, 17, 3, ...)\n  Full diff:\n    [\n  +  ,\n  -  ('00150',\n  -   'Удельный показатель по отказам 1, 2 категории',\n  -   'ед/млн. поездо-км',...\n  \n  ...Full output truncated (289730 lines hidden), use '-vv' to show", "trace": "connection = <cursor object at 0x03B7B128; closed: 0>, hcodes = ('00018', '00154', '00150')\nquery_3_03 = 'create or replace function public.lilo_auto_qa() \\n\\treturns table(\\thcode_id bpchar, hcode_name varchar(300), hcode_....kato_id \\n\\t\\tand a.dir_id = b.dir_id\\ncross join (select * from dm.d_hcode_t where id = %s) hcd\\n    ;\\nend;\\n$f$\\n;'\nquery_3_04 = \"select * from public.lilo_auto_qa()\\nexcept\\nselect  hcode_id, hcode_name, hcode_unit_name, org_id, dor_kod, date_typ...uch_id, nod_id, dir_id, kato_id, vids_id from dm_rep.dm_all_indicators_v\\nwhere hcode_id = %s\\nand calc_rule = 'ratio'\"\n\n    @allure.feature('Расчетные показатели')\n    @allure.story('Полная сверка')\n    @allure.severity('critical')\n    @pytest.mark.parametrize('hcodes',hcodes)\n    def test_empty(connection, hcodes , query_3_03, query_3_04):\n        connection.execute(query_3_03, (hcodes[0], hcodes[1], hcodes[2], hcodes[2]))\n        connection.execute(query_3_04, (hcodes[2],))\n        lines = connection.fetchall()\n        connection.execute('drop function public.lilo_auto_qa();')\n    \n>       assert lines == []\nE       AssertionError: assert [('00150', 'У... 3, ...), ...] == []\nE         Left contains 17043 more items, first extra item: ('00150', 'Удельный показатель по отказам 1, 2 категории', 'ед/млн. поездо-км', 480, 17, 3, ...)\nE         Full diff:\nE           [\nE         +  ,\nE         -  ('00150',\nE         -   'Удельный показатель по отказам 1, 2 категории',\nE         -   'ед/млн. поездо-км',...\nE         \nE         ...Full output truncated (289730 lines hidden), use '-vv' to show\n\ntest_income.py:42: AssertionError"}, "attachments": [{"name": "stdout", "source": "782cdbd2-4819-4969-9cb1-e335d8ef890e-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "hcodes", "value": "('00018', '00154', '00150')"}], "start": 1604502590261, "stop": 1604502621973, "uuid": "f5c1bf2d-029b-4e25-9f55-103ed4e2350f", "historyId": "353939674b94c4c9bb37ef02cf2da03d", "testCaseId": "07ce32e44c681d301be8a13e0c006f7d", "fullName": "test_income#test_empty", "labels": [{"name": "severity", "value": "critical"}, {"name": "feature", "value": "Расчетные показатели"}, {"name": "story", "value": "Полная сверка"}, {"name": "suite", "value": "test_income"}, {"name": "host", "value": "62032-nb"}, {"name": "thread", "value": "1768-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_income"}]}